client/client_pool/src/concord_client_pool.cpp

```diff
@@ -380,6 +375,10 @@ void ConcordClientPool::CreatePool(concord::config_pool::ConcordClientPoolConfig
   }
   batch_timer_ =
       std::make_unique<Timer_t>(timeout, [this](ClientPtr client) -> void { OnBatchingTimeout(std::move(client)); });
+#ifdef USE_COMM_TLS_TCP
```

Comment:
What was the behavior before this ifdef?
This code should be tested with the compiler flags we use in deployment.

Reply:
the same behavior, just moved vars unfer the ifdef, because otherwise compiler complains they are unused